include/sound/sof/info.h

```diff
@@ -31,6 +31,7 @@ enum sof_ipc_ext_data {
 	SOF_IPC_EXT_UNUSED		= 0,
 	SOF_IPC_EXT_WINDOW		= 1,
 	SOF_IPC_EXT_CC_INFO		= 2,
+	SOF_IPC_EXT_PROBE_INFO		= 3,
```

Comment:
@lgirdwood @jajanusz It really would be better to do the kernel PRs at the same time and have the person doing the change write a git commit message that explains the change. This "late-bound" sync of changes to kernel is not fun. Thanks for @xiulipan doing the patch, but I again have to ask for some improvements to the commit message:
- "add support for probe extended data"
- add explanation to commit body about the feature.. you should be able to copy this from the FW commit

Reply:
agree with @kv2019i , this is a complete fail of the ABI change process.

Reply:
The kernel and FW PRs must be done together and mention each other. It is up to maintainers to label as ABI and then to align both PRs using the classifier. So this is several steps taken by at least 3 people and is not being fully processed by everybody.

I am now wondering whether we should now be moving to a ABI minor bump only for every FW release (when the regular FW releases start soon). This would allow the kernel team to plan each merge window in advance to a specific FW version. @lbetlej fyi - regular FW releases would help here.
There is also complexity about bumping ABI in relation to kernel releases and FW releases. i.e. when do we close the ABI minor merge window for new kernel and FW releases (and use the next minor).